core/object/object.cpp

```diff
@@ -53,7 +53,7 @@ struct _ObjectDebugLock {
 	}
 	~_ObjectDebugLock() {
 		Object *obj_ptr = ObjectDB::get_instance(obj_id);
-		if (likely(obj_ptr)) {
+		if (obj_ptr) {
```

Comment:
Did you intend to remove this? If so should it be part of the first commit?

Reply:
No, I didn't. Good catch!